types/src/transaction/mod.rs

```diff
@@ -1278,3 +1301,12 @@ impl TryFrom<Transaction> for SignedTransaction {
         }
     }
 }
+
+pub mod default_protocol {
+    use super::TransactionInfo;
+
+    pub type AccountTransactionsWithProof = super::AccountTransactionsWithProof<TransactionInfo>;
+    pub type TransactionInfoWithProof = super::TransactionInfoWithProof<TransactionInfo>;
+    pub type TransactionListWithProof = super::TransactionListWithProof<TransactionInfo>;
+    pub type TransactionWithProof = super::TransactionWithProof<TransactionInfo>;
+}
```

Comment:
One thought I had was to maybe centralize this to where the default protocol is defined? That way there's one place we have to go to to figure out all the types needed for that particular protocol.

Reply:
The whole code base will be generic and these "::default_protocol::*" things will be deleted in the end.  The specifics of an config we need to figure out about is the members of the ProtocolSpec trait.